apps/mail/app/api/v1/mail/auth/[providerId]/callback/route.ts

```
@@ -22,6 +22,10 @@ export async function GET(
   const driver = await createDriver(providerId, {});
 
   try {
+    // Log the scope parameter from Google's callback
+    const scope = searchParams.get("scope");
+    console.log("Auth callback received with scope:", scope);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)